alvr/client_openxr/src/interaction.rs

```diff
@@ -1066,6 +1081,40 @@ pub fn get_fb_body_tracking_points(
     Vec::new()
 }
 
+pub fn get_bd_motion_trackers(
+    time: Duration,
+    motion_tracker: &MotionTrackerBD,
+) -> Vec<(u64, DeviceMotion)> {
+    let xr_time = crate::to_xr_time(time);
+
+    if let Some(mut trackers) = motion_tracker
+        .locate_motion_trackers(xr_time)
+        .ok()
+        .flatten()
+    {
+        let mut joints = Vec::<(u64, DeviceMotion)>::with_capacity(3);
+
+        let joints_ids = [*BODY_LEFT_FOOT_ID, *BODY_RIGHT_FOOT_ID, *BODY_HIPS_ID];
```

Comment:
The order I wold use as default is first hips, then left and right feet. This is because of games like VRchat, where the minimum tracker configuration is only hips, then you can add feet.